### ISSUE TITLE
Question:adding silex to my app

### ISSUE THREAD

**Issue by develperbayman** [silexlabs/Silex#372]
Im making a Android app for wordpress and want to use silex as the editor is there any way you can do mobile screen ui????
developerbayman@gmail.com

**Comment by lexoyo** [comment]
Hi @develperbayman 
Silex generates standard HTML pages, so you have several solutions to make a mobile app out of it (including [cordova](https://cordova.apache.org/) which is open source)
Also note that there is a "mobile editor" coming soon in Silex, which will make it easy to design for small screens, see this [work in progress demo here](http://silex-editor.herokuapp.com/) (view > mobile editor)

**Comment by develperbayman** [comment]
OMG I LOVE IT!! you are awsome!

-Jeremy

**Comment by develperbayman** [comment]
Yes but the editor it self isnt touch friendly i code on my mobile all the
time and that would be a good feature

-Jeremy

**Comment by lexoyo** [comment]
Oh ok, this is what you mean ?

It is not hard to do from a technical point of view, mainly a new CSS style sheet, perhaps big icons instead of the small shortcut icons... And then there is a little piece of javascript to catch the touchstart/end. 

If one day someone can help with one or the other, we can achieve this

**Comment by develperbayman** [comment]
Actually it would be a challenge to me but i would like to give it a try if
i need help with the api ill give you a hollar.....i love programming

-Jeremy

**Comment by lexoyo** [comment]
OK :)

**Comment by lexoyo** [comment]
Hello
For your information I have tested a solution to support touch screens. It works but I still think that the current look is not appropriate on mobiles and tablets, what do you think?

* [test it here on a touch screen](http://silex-editor.herokuapp.com/)
* the [code I changed to suport touch screens](https://github.com/lexoyo/Silex/commit/4c3fc3f6fa39285fb163f89ac8eaa14a5287d3d2)

**Comment by antosilex** [comment]
je viens de tester sur iphone 5 avec safari ça bug complétement impossible
de faire quoi que ce soit :/

--

Antonin STEPHANY

Chargé de projet, association Silex Labs
Pour recevoir les news de Silex Labs suivez-nous :

   - Facebook <https://www.facebook.com/silexlabs>
   - Twitter <https://twitter.com/silexlabs>
   - Google+ <https://plus.google.com/communities/107373636457908189681>
   - Agenda Silex Labs <http://www.silexlabs.org/events/liste/>
   - Newsletter mensuelle <http://eepurl.com/F48q5>


2015-12-30 12:45 GMT+01:00 Alex Hoyau <notifications@github.com>:

**Comment by develperbayman** [comment]
Oh i must totally disagree its all about the functionality of the program
if it works i can implement zoom in and zoom out its actually really nice
thats why i chose silex editor to embed in my app to work as a companion to
my webstudiox editor wich is different

you see im combining your editor with my wordpress hosting tool .

...but dont worry im planning on sending all the traffic to your site in my
app its just one of many option toward web development And will be a
"online option"
and put your repo mention in the about

Do you have a android? if so ill send you a apk to show you .....give me
some time tho cause i dont want to show it till its more polished but ill
def throw you an example

see the attachment video sorry my internet is slow and silex is doing
something its never done before notice the strange keyboard behavior but
normally you can just zoom in and do stuff minus the dragging of elements
with touch

This is the hardlink my apps been using:

String url = "http://silex-editor.herokuapp.com/";

-Jeremy

**Comment by lexoyo** [comment]
@antosilex thx for the test, I test only on chome and firefox, and it looks like Silex is crazy on safari ios

@develperbayman interesting! You can send me an apk (a.hoyau [at] silexlabs.org) or post a link to an apk here. Also github do not accept files so can you post a link to the video ?

The zoom in/out and scroll is replaced by the drag/drop functionality on 
http://silex-editor.herokuapp.com/

**Comment by develperbayman** [comment]
Iv been using my android strictly for everything and using github is hard
so let's go here I made a slack page

https://silexlabs.slack.com

Everyone is invited

-Jeremy

@antosilex <https://github.com/antosilex> thx for the test, I test only on
chome and firefox, and it looks like Silex is crazy on safari ios

@develperbayman <https://github.com/develperbayman> interesting! You can
send me an apk (a.hoyau [at] silexlabs.org) or post a link to an apk here.
Also github do not accept files so can you post a link to the video ?

The zoom in/out and scroll is replaced by the drag/drop functionality on
http://silex-editor.herokuapp.com/

—
Reply to this email directly or view it on GitHub
<https://github.com/silexlabs/Silex/issues/372#issuecomment-168065899>.

**Comment by lexoyo** [comment]
FYI I did not manage to join the chat room
I really do not like this UI, we can use IRC (freenode / silexlabs)

**Comment by develperbayman** [comment]
Sweet ircs is cool

**Comment by lexoyo** [comment]
FYI now we have moved to slack: https://silexlabsteam.slack.com/

**Comment by ondrejec** [comment]
Great!
But can you invit us ?

**Comment by lexoyo** [comment]
I did not manage to do it in slack... Do you know how to make an open chat with it? Or @clemenchar maybe?

**Comment by singchan** [comment]
should i close this issue?
